docs/integration/social-networks.md

```diff
@@ -8,28 +8,34 @@ You can set up which social network logins are enabled on which domain in `socia
 
 For every social network, you need `app_id` and `app_secret`. Below you have information on where and how to get them. The credentials then need to be set to the corresponding environment variables.
 
+Every social network require URL for backward redirect to you app.
+In Shopsys platform, you have to add url `{eshop_domain}/social-network/login/{type}` where `{eshop_domain}` is your domain and `{type}` is social network name, e.g.: `google`, `facebook`, `seznam`, etc.
+For local development, social networks usually require https.
+
 ### Facebook
 
 -   go to site https://developers.facebook.com/ and log into Facebook account
 -   create [application](https://developers.facebook.com/apps)
 -   in that application, in `App settings` → `Basic` you will find `app_id` and `app_secret`
 
+P.S.: If you are owner of the Facebook application, and you log in with this owner account, email will not come after login, more info: https://github.com/hybridauth/hybridauth/issues/1372#issuecomment-1542447803
```

Comment:
note / warning / info admonition? We usually don't use P.S. in the documentation

Reply:
I'm not sure what exactly the `email will not come after login` means... Like email is not delivered? Or the email address is not returned from the facebook in the data?

Reply:
this was an original note by Peterko, I do not want to dig much deeper into this, I am afraid of Balrog...however, I have changed it to "warning" and rewritten it a little bit in 93de253eeb